docs/website/blog/2023-03-22-deploying-and-using-in-cluster-devfile-registry.md

```diff
@@ -0,0 +1,330 @@
+hence the requirement of having an Ingress Controller and a reachable Ingress domain.
+On OpenShift, it will deploy a [Route](https://docs.openshift.com/container-platform/4.12/networking/routes/route-configuration.html), which will provide an automatic public HTTP URL for accessing the registry.
+
+1. Clone the `registry-support` repository containing the Helm Chart we will deploy:
+
+```shell
+git clone --depth=1 https://github.com/devfile/registry-support
+```
+
+2. Install the [Helm Chart](https://github.com/devfile/registry-support/tree/main/deploy/chart/devfile-registry) into the current cluster.
+
+<Tabs groupId="devfile-registry-helm">
+  <TabItem value="kubernetes" label="Kubernetes">
+
+An Ingress Controller should have been installed with a domain for Ingress resources.
+
+```console
+helm install my-devfile-registry \
+    ./registry-support/deploy/chart/devfile-registry \
+    --set global.ingress.domain=<domain> \
+    --set global.ingress.class=<ingress-class>
+```
+
+<details>
+<summary>Example output:</summary>
+
+```shell
+$ helm install my-devfile-registry \
+    ./registry-support/deploy/chart/devfile-registry \
+    --set global.ingress.domain=$(minikube ip).nip.io \
+    --set global.ingress.class=nginx
+
+NAME: my-devfile-registry
+LAST DEPLOYED: Fri Mar 24 15:50:18 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+
+```
+</details>
+
+  </TabItem>
+  <TabItem value="openshift" label="OpenShift">
+
+```console
+helm install my-devfile-registry \
+    ./registry-support/deploy/chart/devfile-registry \
+    --set global.isOpenShift=true
+```
+
+<details>
+<summary>Example output:</summary>
+
+```shell
+$ helm install my-devfile-registry \
+    ./registry-support/deploy/chart/devfile-registry \
+    --set global.isOpenShift=true    
+         
+NAME: my-devfile-registry
+LAST DEPLOYED: Fri Mar 24 15:54:42 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+
+```
+</details>
+</TabItem>
+</Tabs>
+
+
+## Determining the Devfile Registry URL
+
+<Tabs groupId="devfile-registry-helm">
+  <TabItem value="kubernetes" label="Kubernetes">
+
+On Kubernetes, the Helm Chart installed above will create an Ingress reachable via a DNS domain named as follows: `devfile-registry-<namespace>.<ingressDomain>`.
+
+You can get the actual Host by getting the Ingress Resource, via the following command: `kubectl get ingress <registryName>`.
+
+<details>
+<summary>Example output:</summary>
+
+```shell
+$ kubectl get ingress my-devfile-registry \
+    -o custom-columns='URL:.spec.rules[*].host' \
+    --no-headers
+
+devfile-registry-default.172.20.0.2.nip.io
+```
+
+In this example, the Devfile Registry is now reachable at http://devfile-registry-default.172.20.0.2.nip.io
```

Comment:
Do you think it would be helpful to add a note that it may take 1 or 2 minutes before the address is accessible?